docker-compose.yml

```diff
@@ -155,6 +155,20 @@ services:
       - DISABLED_FEATURES=["downloadEndpoint","functionEndpoint","pdfEndpoint","screencastEndpoint","scrapeEndpoint","statsEndpoint","workspaces"]
       - FUNCTION_ENABLE_INCOGNITO_MODE=true
 
+  e2e:
+    image: cypress/included:12.7.0
+    profiles: [ "e2e" ]
+    container_name: 'ecamp3-e2e'
+    environment:
+      - DISPLAY
+      - APP_BASE_URL=http://localhost:3000
```

Comment:
This shouldn't be needed. The default value for `baseUrl` is already set to `http://localhost:3000` in cypress.config.js

Reply:
[31fe411](https://github.com/ecamp/ecamp3/pull/3326/commits/31fe411ca7b33cf74dc4dc8d99a024fe7b010344)